### ISSUE TITLE
Monaco editor comment text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52432]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Color Contrast,MAS1.4.3
**Repro Steps:**
1. Launch VS Code.
2. Navigate to Help(Alt+H) menu option and select "Interactive Editor Playground" from dropdown.
3. Navigate to editor  where code is written.
4. Open color contrast analyzer and check the ratio.

**Actual:**
Color Contrast ratio is less than 4.5:1  for all the tag lines inside the code.
Same issue observed on other themes also.
**Expected:**
Color Contrast ratio should be greater than 4.5:1.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to see the tags.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**

![accessibility a11y_vscode_interactiveeditorplayground_colorcontrast2](https://user-images.githubusercontent.com/39204909/41651884-4dc1e256-749f-11e8-95e6-59fba747dfb6.png)

**Comment by misolori** [comment]
This only fails on our light theme

**Comment by isidorn** [comment]
This is a general editor issue, not related to welcome, removing tag

**Comment by misolori** [comment]
https://github.com/Microsoft/vscode/commit/c060253db41822e401ea6eafad9dff5e664648da is now in master
